### PR TITLE
Add nighttime time limit at random selection

### PR DESCRIPTION
**Pull request by FXF48-Mila**

### CODE REVIEW

**Comment by FXF48-Mila** [comment]
Hi @leewujung , I opened a new PR because Aditya found that some of the selected files were daytime recordings. Therefore, I added a filter to `combined_df` that filters the `Datetime UTC` to only the nighttime periods. There were also some minor changes in the selection criteria for extreme weather conditions and the final selected dataset. Possibly, I think I can delete the "Verify that no extreme weather conditions were encountered by the audiomoth deployed in the Telephone Field during the month of July" part because there are more Falses in "Has Extreme Weather?" column with the nighttime filter (but I did not delete this part currently). I'm sorry to bother you again. Can you check my code?

**Comment by FXF48-Mila** [comment]
I filter the datetime to midnight (from 8pm - 5am PST which is 03:00 - 12:00 UTC), which is better for finding bat calls. There were also following some minor changes in the selection criteria for extreme weather conditions and the final selected dataset.

File: data/randomization/randomization-with-weather.ipynb [hunk 1]
```diff
@@ -280,9 +280,7 @@
     "print(save_mean['Sum Rain'].min())\n",
```
Comment (leewujung) [inline data/randomization/randomization-with-weather.ipynb:280]: <p>Just an observation: This is pretty interesting. Compare this with the previous same figure (which contains day time hours also), by selecting only the night time files, you can see the proportion of 0 rainfall becomes less.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/uw-echospace/bat-detector-validation/pull/7/discussion/#comment-1390630624'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

**Comment by leewujung** [comment]
<p>@<a href="https://github.com/FXF48-Mila" target="_blank">FXF48-Mila</a>: LGTM! Feel free to merge.</p>

File: data/randomization/randomization-with-weather.ipynb (hunk 1 above)
Comment (FXF48-Mila) [reply data/randomization/randomization-with-weather.ipynb:280]: Yeah, the change in this graph proves that Seattle has more rainfall at night and less rainfall during the day after June!

**Pull request merged**